datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxBackupEndpoint.java

```diff
@@ -37,7 +37,10 @@ public interface SdxBackupEndpoint {
     @ApiOperation(value = "backup the datalake ", produces = MediaType.APPLICATION_JSON, nickname = "backupDatalake")
     SdxBackupResponse backupDatalakeByName(@PathParam("name") String name,
             @QueryParam("backupLocation") String backupLocation,
-            @QueryParam("backupName") String backupName);
+            @QueryParam("backupName") String backupName,
+            @QueryParam("skipAtlas") boolean skipAtlas,
+            @QueryParam("SkipRangerAudits") boolean skipRangerAudits,
+            @QueryParam("SkipRagerMetadata") boolean skipRangerMetadata);
```

Comment:
have you considered to use request object instead of several query params? also last two query param is not starting with lowercase letter in `@QueryParam` annotation

Reply:
For the API since it's just booleans, it's pretty simple to keep flat. Will extend like this out to CLI as well.